core/templates/dev/head/components/profileLink.js

```diff
@@ -0,0 +1,21 @@
+oppia.directive('profileLinkText', [function() {
+  return {
+    restrict: 'E',
+    scope: {
+      linkedName: '@'
```

Comment:
I was concerned that calling it username might get confused with the username of the person who's currently logged in-- though I suppose if it's limited to just this directive's cope that makes sense. Will change!